package/MDAnalysis/analysis/bat.py

```diff
@@ -0,0 +1,439 @@
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v21 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+r"""Bond-Angle-Torsion coordinates analysis --- :mod:`MDAnalysis.analysis.bat`
+===========================================================================
+
+:Author: David Minh
+:Year: 2020
+:Copyright: GNU Public License, v2 or any higher version
+
+.. versionadded:: N/A
+
+This module contains classes for interconverting between Cartesian and an
+internal coordinate system, Bond-Angle-Torsion (BAT) coordinates [Chang2003]_,
+for a given set of atoms or residues. This coordinate system is designed
+to be complete, non-redundant, and minimize correlations between degrees
+of freedom. Complete and non-redundant means that for N atoms there will
+be 3N Cartesian coordinates and 3N BAT coordinates. Correlations are
+minimized by using improper torsions [Hikiri2016]_.
+
+More specifically, bond refers to the bond length, or distance between
+a pair of bonded atoms. Angle refers to the bond angle, the angle between
+a pair of bonds to a central atom. Torsion refers to the torsion angle.
+For a set of four atoms a, b, c, and d, a torsion requires bonds between
+a and b, b and c, and c and d. The torsion is the angle between a plane
+containing atoms a, b, and c and another plane containing b, c, and d.
+For a set of torsions that share atoms b and c, one torsion is defined as
+the primary torsion. The others are defined as improper torsions, differences
+between the raw torsion angle and the primary torsion. This definition reduces
+the correlation between the torsion angles.
+
+Each molecule also has six external coordinates that define its translation and
+rotation in space. The three Cartesian coordinates of the first atom are the
+molecule's translational degrees of freedom. Rotational degreres of freedom are
+specified by the axis-angle convention. The rotation axis is a normalized vector
+pointing from the first to second atom. It is described by the polar angle, phi,
+and azimuthal angle, theta. omega is a third angle that describes the rotation
```

Comment:
You can use math
```suggestion
pointing from the first to second atom. It is described by the polar angle, :math:`phi`,
and azimuthal angle, :math:`theta`. :math:`omega` is a third angle that describes the rotation
```

You can also include full equations such as 
```reST
.. math::

    \cos\alpha = \frac{\matbf{a}\cdot\mathbf{b}}{ab}
```